packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -917,8 +919,11 @@ test.describe('Content negotiation', () => {
 	test('use:enhance uses action, not POST handler', async ({ page }) => {
 		await page.goto('/routing/content-negotiation');
 
-		page.click('button:has-text("Submit")');
-		await page.waitForResponse('/routing/content-negotiation');
+		await Promise.all([
```

Comment:
it might be worth adding a helper for this too? it looks like there are other tests where we submit forms. e.g. https://github.com/sveltejs/kit/blob/296e850ae761865a6e09beb59750ff6fa59f2d17/packages/kit/test/apps/basics/test/test.js#L1873